kalitectl.py

```diff
@@ -400,7 +400,7 @@ def start_watchify():
     sys.stdout.write('Starting watchify')
 
     watchify_process = subprocess.Popen(
-        ['node compile_javascript.js --debug --watch --staticfiles'],
+        args='node compile_javascript.js --debug --watch --staticfiles',
```

Comment:
[This is recommended.](https://docs.python.org/2/library/subprocess.html#subprocess.Popen) Old version fails on Windows.